scripts/README.md

```diff
@@ -1,8 +1,34 @@
-# Scripts
+- [Setup](#setup)
```

Comment:
Can we update the TOC automatically when changes to the readme are made?

Reply:
I've been using a VS Code plugin called `Markdown All In One` to generate it. I'll see if there are other options for generating as well.

Reply:
Intellij's Markdown plugin also has a generate Table of Contents feature. In Intellij, you open the markdown file -> command + n -> Generate Table of Contents